docs/config.json

```diff
@@ -881,6 +881,10 @@
             {
               "title": "Applications",
               "slug": "/machine-id/guides/applications/"
+            },
+            {
+              "title": "Host Certificates",
+              "slug": "/machine-id/guides/hostcertificate/"
```

Comment:
We should probably make this `/machine-id/guides/host-certificate` for consistency with the rest of the docs.